openmetadata-ui/pom.xml

```diff
@@ -137,6 +137,20 @@
               <!--arguments>build</arguments-->
             </configuration>
           </execution>
+          <execution>
+            <id>yarn run build-test</id>
+            <goals>
+              <goal>yarn</goal>
+            </goals>
+            <!-- optional: the default phase is "generate-resources" -->
+            <phase>test</phase>
+            <configuration>
+              <!-- optional: if not specified, it will run gulp's default task
```

Comment:
Can we remove these comments?

They seem outdated

For reference on what these parameters are, one can always look at maven documentation

Reply:
Addressed in the latest commit.